db/seeds.rb

```diff
@@ -8,16 +8,20 @@
 require "open-uri"
 
 puts 'Cleaning the database...'
+FlashcardTemplate.destroy_all
```

Comment:
add has_many :flashcard_templates, dependent: :destroy to flashcardset model
then you can delete this line